templates/miq-template.yaml

```diff
@@ -108,15 +108,15 @@ objects:
           imagePullPolicy: IfNotPresent
           name: manageiq
           livenessProbe:
-            httpGet:
-              path: /
-              port: 80
+            tcpSocket:
+              port: 443
             initialDelaySeconds: 480
             timeoutSeconds: 3
           readinessProbe:
             httpGet:
               path: /
-              port: 80
+              port: 443
+              scheme: HTTPS
```

Comment:
@fbladilo  please remind me the readiness probe is only about connecting it to the service (and will not reschedule the pod) right ?